wpeverest/everest-forms
# Feature - Elementor Widget

### All Submissions:



### Changes proposed in this Pull Request:
Previously user have to search for elementor short code widgets in order to display forms. This PR gives  the everest form elementor widget to display form directly.


### How to test the changes in this Pull Request:

1. Download and Install elementor plugin from wordpress.org.
2. Goto pages and click edit with elementor.
3. Goto Basic Section and search everest form widget.
4. Drag the everest form widget and select the form verify it is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Elementor Widget.
